source/navitiacommon/navitiacommon/bin.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+import logging
+import re
+import subprocess
+import sys
+
+"""
+Function about binary
+"""
+
+logger = logging.getLogger(__name__)  # type: logging.Logger
+
+
+def run(exec_name, args, new_logger=None):
```

Comment:
This change is way outside the scope of what the PR says...  (-> `eitri: improve logs`).
You break the logs integration with `Tyr` and break it's asynchronous tasks here. 
Please, don't change anything that might impact anything else than `eitri` here.

Reply:
You're right, I check this and extract it from this patch to do an other PR.